devops/scripts/benchmarks/benches/compute.py

```diff
@@ -332,7 +343,11 @@ def name(self):
         # to match the existing already stored results
         events_str = " not using events" if not self.UseEvents else ""
 
-        return f"api_overhead_benchmark_{self.runtime.value} SubmitKernel {order}{completion_str}{events_str}"
+        kernel_exec_time_str = (
+            f" KernelExecTime=self.KernelExecTime" if self.KernelExecTime != 1 else ""
```

Comment:
{self.KernelExecTime} in f-string. Same below

Reply:
Done.